imports/plugins/included/surcharges/server/no-meteor/xforms/xformSurchargeMessage.js

```diff
@@ -0,0 +1,13 @@
+import getSurchargeMessageForLanguage from "../util/getSurchargeMessageForLanguage";
+
+/**
+ * @name xformSurchargeMessage
+ * @summary Returns surcharge message in specified language
+ * @param {String} language Language to filter items by
+ * @param {Array} messagesByLanguage Array to check language against
+ * @return {String} Translated message to return to client
+ */
+export default async function xformSurchargeMessage(language, messagesByLanguage) {
+  const translatedMessage = getSurchargeMessageForLanguage(language, messagesByLanguage);
```

Comment:
Looks like `xformSurchargeMessage` args and return value are identical to `getSurchargeMessageForLanguage` and so providing no additional value. You could delete it and call `getSurchargeMessageForLanguage` directly.

Reply:
Good call. This wasn't the case originally, more was being done in xformSurchargeMessage before we updated the language schema.